docs/administration/parameters.md

```diff
@@ -52,6 +52,7 @@ This option can be deactivated to have a permanent announcement.
 
     [Please read the dedicated page to have more information](enterprise.md)
 
+<<<<<<< HEAD
```

Comment:
Isn't that a residue of conflict to be removed?

Reply:
Yeah! I haven't seen it. I will compare with the version we have in the main branch and make sure everything is Ok